test/integration/graql/reasoner/ReasoningIT.java

```diff
@@ -577,129 +136,4 @@ public void whenAppendingRolePlayers_noNewRelationsAreCreated() {
-            try (Transaction tx = session.transaction(Transaction.Type.READ)) {
-                List<ConceptMap> derivedRPTriples = tx.execute(Graql.<GraqlGet>parse("match (inferredRole: $x, inferredRole: $y, inferredRole: $z) isa derivedRelation; get;"));
-                List<ConceptMap> derivedRelations = tx.execute(Graql.<GraqlGet>parse("match $r (inferredRole: $x, inferredRole: $y, inferredRole: $z) isa derivedRelation; get;"));
-
-                //NB: same answer is obtained from both rules
-                //three symmetric roles hence 3! results
-                assertEquals("Rule body is not rewritten correctly!", 6, derivedRPTriples.size());
-                assertEquals("Rule body is not rewritten correctly!", 6, derivedRelations.size());
-                assertEquals(1, derivedRelations.stream().map(ans -> ans.get("r")).collect(Collectors.toSet()).size());
-            }
-        }
-    }
-
-    @Test //tests whether shared resources are recognised correctly
-    public void inferrableRelationWithRolePlayersSharingResource() {
-        try (Session session = server.sessionWithNewKeyspace()) {
-            loadFromFileAndCommit(resourcePath, "testSet29.gql", session);
-            try (Transaction tx = session.transaction(Transaction.Type.READ)) {
-
-                String queryString = "match " +
-                        "(role1: $x, role2: $y) isa binary-base;" +
-                        "$x has name $n;" +
-                        "$y has name $n;" +
-                        "get;";
-
-                String queryString2 = "match " +
-                        "(role1: $x, role2: $y) isa binary-base;" +
-                        "$x has name $n;" +
-                        "$y has name $n;" +
-                        "$n == 'a';" +
-                        "get;";
-
-                String queryString3 = "match " +
-                        "(role1: $x, role2: $y) isa binary-base;" +
-                        "$x has name 'a';" +
-                        "$y has name 'a';" +
-                        "get;";
-
-                List<ConceptMap> answers = tx.execute(Graql.parse(queryString).asGet());
-                List<ConceptMap> answers2 = tx.execute(Graql.parse(queryString2).asGet());
-                List<ConceptMap> answers3 = tx.execute(Graql.parse(queryString3).asGet());
-
-                assertEquals(3, answers.size());
-                answers.forEach(ans -> {
-                    assertEquals(3, ans.size());
-                    assertEquals(ans.get("x"), ans.get("y"));
-                });
-
-                assertEquals(1, answers2.size());
-                assertEquals(1, answers3.size());
-                answers2.stream()
-                        .map(a -> a.project(Sets.newHashSet(new Variable("x"), new Variable("y"))))
-                        .forEach(a -> assertTrue(answers3.contains(a)));
-            }
-        }
-    }
-
-    @Test
-    public void ternaryRelationsRequiringDifferentMultiunifiers() {
```

Comment:
already exists in variable-role

Reply:
lets go over this one  - there's a reason these are all grouped together!

Reply:
They are also grouped together in variable-roles.feature, in the scenario:
`Scenario: variable roles are correctly mapped to answers for a ternary relation with 3 possible roleplayers`